crates/http/src/trigger.rs

```diff
@@ -63,6 +70,7 @@ impl HandlerType {
                     `{WASI_HTTP_EXPORT_2023_10_18}`, \
                     `{WASI_HTTP_EXPORT_2023_11_10}`, \
                     `{WASI_HTTP_EXPORT_0_2_PREFIX}.*`, \
+                    `{WASI_HTTP_EXPORT_0_3_PREFIX}.*`, \
```

Comment:
Should this also use the snapshot approach we used for the p2 snapshots? I don't think we want to commit to making the snapshots backwards compatible among each other, after all.